src/main/java/it/pagopa/pn/deliverypush/service/impl/TimeLineServiceImpl.java

```diff
@@ -99,7 +112,7 @@ public boolean addTimelineElement(TimelineElementInternal dto, NotificationInt n
 
                 MDC.remove(MDCUtils.MDC_PN_CTX_TOPIC);
 
-                return timelineInsertSkipped;
+                return !timelineInsertSkipped;
```

Comment:
hmmmm cambiare sta condizione ha impatti in giro per il codice, e dai file modificati  non son stati guardati...
meglio lasciarla come prima valà! :)